### PR TITLE
improved logging by changing eprint to info in strace.rs

### PR DESCRIPTION
**Pull request by sagar10arya**
in response to #370  
improved logging by replacing eprint! macro to info! in veracruz-main\execution-engine\src\wasi\strace.rs

### CODE REVIEW

**Review by ShaleXIONG — APPROVED** [review]

**Comment by dominic-mulligan-arm** [comment]
@sagar10arya you need to add a `log` import to `execution-engine/src/Cargo.toml`.  Also, as a project we tend to avoid merges in favour of rebasing.  You can change the behaviour of your Git client to ensure that rebasing is always preferred by setting:

```
git config --global pull.rebase true
```

File: execution-engine/Cargo.toml [hunk 1]
```diff
@@ -24,6 +24,8 @@ std = [
 ]
 
 [dependencies]
+use  log :: info
```
Comment (ShaleXIONG) [inline execution-engine/Cargo.toml:27]: It should be in the syntax of `log = "0.4.13"`, meaning, import log crate (or package) of version 0.4.13

File: execution-engine/Cargo.toml [hunk 2]
```diff
@@ -24,6 +24,8 @@ std = [
 ]
 
 [dependencies]
+log = "0.4.13"
+
```
Comment (ShaleXIONG) [inline execution-engine/Cargo.toml:28]: Delete the extra blank line.

Comment (sagar10arya) [reply execution-engine/Cargo.toml]: > Delete the extra blank line.

Done.

**Comment by dominic-mulligan-arm** [comment]
@sagar10arya can you please configure your Git client to prefer rebasing rather than merging when pulling changes?  The instructions are in the thread, up above.  Thanks!

**Pull request merged**